ArcadiaParties.API/Controllers/UsersController.cs

```diff
@@ -50,10 +53,19 @@ public async Task<IActionResult> GetUsers(CancellationToken cancellationToken)
         [Route("GetCurrentUser")]
         public async Task<IActionResult> GetCurrentUser(CancellationToken cancellationToken)
         {
-            var query = new GetCurrentUserQuery(User);
-            var user = await _mediator.Send(query, cancellationToken);
+            var userQery = new GetCurrentUserQuery(User);
```

Comment:
You don't need whole user from your database, only roles.

Reply:
Fixed